### PR TITLE
Added Os.ark and home_dir function

### PR DESCRIPTION
**Pull request by Wafelack**
- Added a home_dir function (not cross platform at all for now) displaying current user home dir

### CODE REVIEW

File: Os.ark [hunk 1]
```diff
@@ -0,0 +1,14 @@
+###
+# @meta Os
```
Comment (SuperFola) [inline Os.ark:2]: rename to `os`

File: Os.ark [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# @brief Returns the home dir of the current user
+# @author https://github.com/Wafelack
+###
+(let os:home_dir (fun () {
+  (sys:exec "whoami > whoami.txt")
+  (let whoami (io:readFile "whoami.txt"))
+  (mut res "")
```
Comment (SuperFola) [inline Os.ark:9]: remove, it isn't used

File: Os.ark (hunk 1 above)
Comment (SuperFola) [inline Os.ark:2]: rename to OS or os according to the new filename

File: Os.ark [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  (if (= whoami "root\n" )
+    (+ "" "/")
+    # else
+    ( + "/home/" whoami))
```
Comment (SuperFola) [inline Os.ark:13]: removing useless white space between `(` and `+`

**Review by SuperFola — CHANGES_REQUESTED** [review]
either rename to OS.ark or os.ark, Os.ark is weird because it's Operating System and both words are important

File: Os.ark (hunk 2 above)
Comment (Wafelack) [reply Os.ark:9]: What to remove ? The whole 4 lines ?

Comment (Wafelack) [reply Os.ark:9]: Ok no i finally understood

**Review by SuperFola — APPROVED** [review]

**Comment by SuperFola** [comment]
TODO for later
- add on the README.md
- update later when we'll have a better way to retrieve the output of a sys call
- make it work on linux and windows

**Pull request merged**